src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -243,15 +241,8 @@ function ReportPreview({
     }
 
     const isArchived = isArchivedReport(iouReport);
-    const currentUserAccountID = getCurrentUserAccountID();
     const isAdmin = policy?.role === CONST.POLICY.ROLE.ADMIN;
-    const shouldShowSubmitButton =
-        isOpenExpenseReport &&
-        !isArchived &&
```

Comment:
I don't think `canSubmitReport` has `!isArchived` in the condition. That will make user be able to submit while workspace was deleted.

<img width="1135" alt="Screenshot 2025-01-15 at 13 31 01" src="https://github.com/user-attachments/assets/4c01f98f-d586-441b-9eca-37ca70bfc8d8" />